tests/test_forgejo/test_misc.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2024-2025 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from webhook_to_fedora_messaging_messages.forgejo import ForgejoMessageV1
```

Comment:
The tests in this file are only testing `ForgejoMessageV1`, I think they should be in `test_forgejo.py `.